molecule/pxc/pxc80-common-upgrade/.yamllint

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
Do we need yamllint files (all of them) here? Please remove if we do not use them.